lib/AST/NameLookupRequests.cpp

```diff
@@ -156,6 +156,9 @@ HasMissingDesignatedInitializersRequest::evaluate(Evaluator &evaluator,
   if (!scope.isPublic())
     return false;
 
+  // Make sure any implicit constructors are synthesized.
+  (void)subject->getAllMembers();
```

Comment:
In other places I see us doing this:

```
  // Make sure we've resolved implicit members, if we need them.
  namelookup::installSemanticMembersIfNeeded(type, name);
```

This ultimately calls `TypeChecker::addImplicitConstructors()` which does less work than `getAllMembers()`.

Reply:
Good catch, it looks like `ResolveImplicitMemberRequest` is probably what I ought to use here.